tests/wrapper/doesnt_clobber_env.fish

```diff
@@ -1,5 +1,6 @@
 set --global --export SHELL /bin/sh
-# don't set FZF_DEFAULT_OPTS so it will set one
+# erase FZF_DEFAULT_OPTS so that one will be set by the wrapper function
+set --erase FZF_DEFAULT_OPTS
```

Comment:
out of scope change to facilitate local tests where I do have a FZF_DEFAULT_OPTS set already